modules/osd-create-cluster-rhm-gcp-account.adoc

```diff
@@ -90,22 +90,37 @@ After your cluster is created, you can change the number of compute nodes, but y
 
 . Click *Next*.
 
-. In the *Cluster privacy* dialog, select *Public* or *Private* to use either public or private API endpoints and application routes for your cluster.
+. In the *Cluster privacy* dialog, select *Public* or *Private* to use either public or private API endpoints and application routes for your cluster. If you select *Private*, the optional *Use Private Service Connect* feature is automatically enabled.
+If you do not want to use Private Service Connect (PSC), disable by clicking the *Use Private Service Connect* checkbox.
++
+[NOTE]
+====
+Red Hat recommends Private Service Connect (PSC) for network connectivity.
```

Comment:
```suggestion
Red Hat recommends using PSC for more secure network connectivity.
```
Brain not fully braining here again but don't feel it's strong enough in it's current iteration. An alternative suggestion would be: PSC is Red Hat's recommended form of network connectivity.

Reply:
Brain not Braining! :)